### PR TITLE
Delivery for Manager Instant App - UAT- AGOL-2024.R02

### PR DESCRIPTION
**Pull request by sumitzarkar**
## Implemented 
- #417 
- https://github.com/Esri/solutions-components/issues/608#issuecomment-2110793075
- https://github.com/Esri/solutions-components/issues/607#issuecomment-2122271466
- https://github.com/Esri/solutions-components/issues/308#issuecomment-2122482896
- Updated demos to use the latest css for calcite components

### CODE REVIEW

File: src/components/layer-table/layer-table.tsx [hunk 1]
```diff
@@ -1797,15 +1787,26 @@ export class LayerTable {
...
+    //get the sortField and sortOrder from the configuration
     if (this.mapInfo?.layerOptions?.layers?.length > 0) {
-      const configuredLayer = this.mapInfo.layerOptions?.layers.filter((layer) => layer.id === this._layer.id);
-      sortField = configuredLayer[0]?.fields?.includes(this.sortField) ? this.sortField : this._layer.objectIdField;
+      configuredLayer = this.mapInfo.layerOptions.layers.filter((layer) => layer.id === this._layer.id);
+      if (configuredLayer && configuredLayer.length > 0) {
+        configuredLayer = configuredLayer[0];
+        //if sort field is defined and sortField is available in the fields then use it 
+        if (configuredLayer.sortField && configuredLayer.fields?.includes(configuredLayer.sortField)) {
+          sortField = configuredLayer.sortField
```
Comment (jmhauck) [inline src/components/layer-table/layer-table.tsx:1804]: add `;`

File: src/components/layer-table/layer-table.tsx [hunk 2]
```diff
@@ -1797,15 +1787,26 @@ export class LayerTable {
...
+        }
+        //use sort order if configured
+        sortOrder = configuredLayer?.sortOrder ? configuredLayer.sortOrder : "desc"
```
Comment (jmhauck) [inline src/components/layer-table/layer-table.tsx:1807]: add `;`

**Review by jmhauck — APPROVED** [review]

**Comment by sumitzarkar** [comment]
@jmhauck - added semi-colon

**Comment by jmhauck** [comment]
thanks @sumitzarkar

**Review by jmhauck — APPROVED** [review]

**Pull request merged**